tools/StaticAnalysis/Exceptions/AzureRM.Compute/BreakingChangeIssues.csv

```diff
@@ -1,2 +1,22 @@
 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
-"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMDataDiskCommand","Add-AzureRmVMDataDisk","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Add-AzureRmVMDataDisk' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Add-AzureRmVMDataDisk'."
\ No newline at end of file
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMDataDiskCommand","Add-AzureRmVMDataDisk","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Add-AzureRmVMDataDisk' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Add-AzureRmVMDataDisk'."
```

Comment:
Whats with all these breaking changes?

Reply:
When I generate a client library with the latest AutoRest, the generated object is changed from string to enum.   (There was no Swagger spec change.)  However, the generated client library accepts string for enum, thus there is no breaking change for any method, except the type of parameters of a return type.

Reply:
@hyonholee  sounds good.